schemas/test/en/test_anchor_scrolling.json

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
I don't think we need to add a new schema here, we have a number that have multiple answers, test_numbers would be a good one to use. If you really did want to keep the schema, it should be stripped back to the basics of the test (i.e remove the skip conditions/extra blocks etc) The description would also need updating

Reply:
I don't think test_numbers would be suitable as there's no final summary. I'll reuse the one I have

Reply:
Had a look at the latest iteration, and I'm still not convinced, we need this. Can we not reuse, `test_section_summary`, this seems almost identical?. You just need to do the same edit and add an extra answer in, which shouldn't cause much issue with existing tests?.

Reply:
It is almost identical, but it obviously would mean the tests include a lot of elements aren't needed for the test. This way it's at least streamlined.

Reply:
Okay fair enough.